Cargo.toml

```diff
@@ -64,7 +64,7 @@ batch = ["alloc", "merlin", "rand_core"]
 fast = ["curve25519-dalek/precomputed-tables"]
 digest = ["signature/digest"]
 # This features turns off stricter checking for scalar malleability in signatures
-legacy_compatibility = []
+legacy_compatibility = ["curve25519-dalek/legacy_compatibility"]
```

Comment:
`Scalar::from_bits` is necessary when we have bad scalar checks. This is why I had to make the `curve25519-dalek/legacy_compatibility` feature